src/transaction/log_manager.c

```diff
@@ -1736,7 +1750,7 @@ log_final (THREAD_ENTRY * thread_p)
       return;
     }
 
-  if (log_Gl.append.vdes == NULL_VOLDES)
+  if (!log_Log_header_initialized)
```

Comment:
I don't like, nor do I fully understand this but it works.
Tested:
- createdb, deletedb
- normal mode server execution
- TSRS/PS execution